libpsterra/pom.xml

```diff
@@ -63,22 +76,23 @@
             <version>1.1.0</version>
         </dependency>
 
-<!-- Spigot -->
+        <!-- Paper -->
         <dependency>
-            <groupId>org.spigotmc</groupId>
-            <artifactId>spigot-api</artifactId>
-            <version>1.12.2-R0.1-SNAPSHOT</version>
+            <groupId>io.papermc.paper</groupId>
+            <artifactId>paper-api</artifactId>
+            <version>1.20.1-R0.1-SNAPSHOT</version>
```

Comment:
Is there any reason why you are still using 1.20.1? The plotsystem has since been updated to 1.21.4 and we dont really plan on supporting older versions as that would make things much more complicated

Reply:
You are right, this was from before you updated the PlotSystem to 1.21.4.
I've committed the new paper version as suggested.